spec/mailer/planning_application_mailer_spec.rb

```diff
@@ -15,16 +15,23 @@
 
   let!(:reviewer) { create :user, :reviewer, local_authority: local_authority }
   let!(:assessor) { create :user, :assessor, local_authority: local_authority }
+
   let!(:planning_application) do
     create(
       :planning_application,
       :determined,
       agent_email: "cookie_crackers@example.com",
       applicant_email: "cookie_crumbs@example.com",
       local_authority: local_authority,
-      decision: "granted"
+      decision: "granted",
+      address_1: "123 High Street", # rubocop:disable Naming/VariableNumber
```

Comment:
At some point maybe we should decide whether to rename the column or disable the rule globally?